fleet/view/js/fleet.js

```diff
@@ -0,0 +1,316 @@
+var loc = window.location.pathname;
+var split_url = loc.split('/');
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/davidgunadi/hello-world/pullRequest?prid=8400071)
```suggestion
var split_url = loc.split("/");
```